job_offers_data_load/PostgreSQL_data_insert.py

```diff
@@ -0,0 +1,94 @@
+import pandas
+import sqlalchemy.orm
+import sqlalchemy.sql.schema
+from sqlalchemy import select
+import datetime
+from job_offers_data_load.PostgreSQL_tables_declaration import *
+
+
+def is_already_for_list_like(dataframe: pandas.DataFrame,
+                             iterator: int,
+                             dataframe_column_name: str,
+                             session: sqlalchemy.orm.session.Session,
+                             schema_from: sqlalchemy.sql.schema,
+                             schema_from_column) -> list:
+    """Returns list that will be used for creation of Table. Makes sure there will be no duplicates."""
+    temp_list = []
+    for item in dataframe.loc[iterator, dataframe_column_name]:
+        statement = select(schema_from).where(schema_from_column == item)
+        is_already = session.execute(statement).fetchone()
+        if is_already is None:
+            temp_list.append(schema_from(item))
+        else:
+            temp_list.append(is_already[0])
+    return temp_list
+
+
+def stamp_expired(data_df: pandas.DataFrame,
```

Comment:
What is `data_df`? I'd recomend to somehow explain what you have in your dataframe.
Because now it looks like `lst: list` or `dct: dict`